sdk/advisor/arm-advisor/README.md

```diff
@@ -2,11 +2,15 @@
 
 This package contains an isomorphic SDK (runs both in Node.js and in browsers) for AdvisorManagementClient.
 
+> ⚠️ This package @azure/arm-advisor with versions lower than 2.0.0 are going to be deprecated in March 2022, we strongly recommend you to upgrade your dependency on it to version 2.0.0 or above as soon as possible. The deprecate means, it starts the end of support for that library. You can continue to use the libraries indefinitely (as long as the service is running), but after 1 year, no further bug fixes or security fixes will be provided.
```

Comment:
> with versions lower than 2.0.0 are going to be deprecated in March 2022, we strongly recommend you to upgrade your dependency on it to version 2.0.0 or above as soon as possible.

Not all Track 1 packages are version 2, some are of higher version

Reply:
yeah the version here is RP specific

Reply:
@colawwj Please update accordingly.